src/patient-lab-dashboard/table/pending-lab-orders/pending-lab-orders.tsx

```diff
@@ -119,19 +119,7 @@ const PendingLabOrdersTable = ({
                 <Table title="lab-order-table" useZebraStyles={true}>
                   <TableHead>
                     <TableRow>
-                      <TableSelectAll
-                        {...getSelectionProps()}
-                        onSelect={() => {
-                          if (selectedPendingOrder.length > 0 && !selectAll) {
-                            setSelectAll(false)
-                            setSelectedPendingOrder([])
-                          } else {
-                            setSelectAll(!selectAll)
-                          }
-                        }}
-                        disabled={onButtonClick || onEnterResultButtonClick}
-                        checked={selectAll}
-                      />
+                      <TableHeader></TableHeader>
```

Comment:
Why is there a empty <TableHeader> tag? If this is not required can we remove this?

Reply:
We do not want `Select All` checkbox at the top header but wanted to keep that column thus kept it.